index.bs

```diff
@@ -805,6 +805,57 @@ and doing cleanup as necessary (see above).
   is still [under discussion](https://github.com/whatwg/html/issues/6588).
 </div>
 
+<h3 id="third-party-tools">Handling conflicts with third-party tools</h3>
```

Comment:
We should be consistent in tone with the rest of the document. e.g. "Ensure x with third-party tools"

Reply:
"Avoid Designing Around Conflicts with Third Party Tools"?
"Priotise Ussability over Compatibility with Third Party Tools"